### ISSUE TITLE
Installation fails on daily build of elementaryOS 6 / Odin

### ISSUE THREAD

**Issue by carniz** [pantheon-tweaks/pantheon-tweaks#43]
This might be a an issue caused by the fact that Odin hasn't been released yet, but I thought it might be worth reporting it anyway.

```
mikael@elementary-os:~$ sudo add-apt-repository ppa:philip.scott/pantheon-tweaks
Traceback (most recent call last):
  File "/usr/bin/add-apt-repository", line 108, in <module>
    sp = SoftwareProperties(options=options)
  File "/usr/lib/python3/dist-packages/softwareproperties/SoftwareProperties.py", line 118, in __init__
    self.reload_sourceslist()
  File "/usr/lib/python3/dist-packages/softwareproperties/SoftwareProperties.py", line 613, in reload_sourceslist
    self.distro.get_sources(self.sourceslist)    
  File "/usr/lib/python3/dist-packages/aptsources/distro.py", line 91, in get_sources
    raise NoDistroTemplateException(
aptsources.distro.NoDistroTemplateException: Error: could not find a distribution template for Elementary/next
```

The Elementary/next error is probably caused by VERSION_CODENAME=next in /etc/os-release:

```
mikael@elementary-os:~$ cat /etc/os-release 
NAME="elementary OS"
VERSION="6.0 Next"
ID=elementary
ID_LIKE=ubuntu
PRETTY_NAME="elementary OS 6.0 Next"
LOGO=distributor-logo
VERSION_ID="6.0"
HOME_URL="https://elementary.io/"
DOCUMENTATION_URL="https://elementary.io/docs/learning-the-basics"
SUPPORT_URL="https://elementary.io/support"
BUG_REPORT_URL="https://github.com/elementary/os/issues/new"
PRIVACY_POLICY_URL="https://elementary.io/privacy-policy"
VERSION_CODENAME=next
UBUNTU_CODENAME=focal
```

After creating a /etc/apt/sources.list.d/pantheon-tweaks.list file manually containing `deb http://ppa.launchpad.net/philip.scott/elementary-tweaks/ubuntu focal main`, I get these warnings+error when running `sudo apt update`:

```
mikael@elementary-os:~$ sudo apt update
Hit:1 http://ppa.launchpad.net/elementary-os/daily/ubuntu focal InRelease
Get:2 http://mirrors.ubuntu.com/mirrors.txt Mirrorlist [356 B]
Get:5 http://ppa.launchpad.net/philip.scott/elementary-tweaks/ubuntu focal InRelease [18.1 kB]
Err:5 http://ppa.launchpad.net/philip.scott/elementary-tweaks/ubuntu focal InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY C1D89326B1C71AB9
Hit:6 http://ppa.launchpad.net/elementary-os/os-patches/ubuntu focal InRelease
Hit:3 http://ftp.acc.umu.se/ubuntu focal InRelease                     
Hit:4 https://ftpmirror1.infania.net/ubuntu focal-updates InRelease
Reading package lists... Done                             
W: GPG error: http://ppa.launchpad.net/philip.scott/elementary-tweaks/ubuntu focal InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY C1D89326B1C71AB9
E: The repository 'http://ppa.launchpad.net/philip.scott/elementary-tweaks/ubuntu focal InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

**Comment by ryonakano** [comment]
I can confirm this issue but this is due to patching issue in elementary OS (see https://github.com/elementary/os-patches/issues/136). We can't take action in the scope of Pantheon Tweaks so I'm closing this as out of scope.

**Comment by ryonakano** [comment]
Leaving a temporary solution to install Pantheon Tweaks on elementary OS 6. Again, it's not supported to install Pantheon Tweaks on elementary OS 5.x using PPA.

```bash
sudo apt-key adv --keyserver keyserver.ubuntu.com --recv c42d52715a84c7d0d02fc740c1d89326b1c71ab9
echo -e "deb http://ppa.launchpad.net/philip.scott/pantheon-tweaks/ubuntu focal main\ndeb-src http://ppa.launchpad.net/philip.scott/pantheon-tweaks/ubuntu focal main" | sudo tee /etc/apt/sources.list.d/pantheon-tweaks.list
sudo apt update
sudo apt install -y pantheon-tweaks
```

**Comment by pporkhun** [comment]
after a while elementary-tweaks getting not working on my eOS6, I purge and install pantheon-tweaks. Icon become on system settings but it is not starting. How can I trace error and fix it?

**Comment by ryonakano** [comment]
Hi @pporkhun, thank you for your report.
You mean, openin Tweaks pane closes System Settings immediately, right? I assume it might be because of #65, but I'n not certain yet. Would you open Terminal, type `io.elementary.switchboard`, press Enter, and open Tweaks pane? If the following error message is displayed, then your issue is the same with #65 and it will be fixed by #66.

```
(io.elementary.switchboard:19342): GLib-GIO-ERROR **: 18:28:20.571: Settings schema 'io.elementary.files.preferences' does not contain a key named 'single-click'
```

**Comment by pporkhun** [comment]
in my copy it looks like:
when i took in terminal command 
io.elementary.files.preferences
answer is:
io.elementary.files.preferences: command not found

**Comment by ryonakano** [comment]
Hi @pporkhun, thank you for your reply. Looking at your log, your issue is the same with #65. I already made a pull request to fix it (#66) so will be resolved soon.
